src/pydata_sphinx_theme/assets/styles/components/_switcher-version.scss

```diff
@@ -1,6 +1,11 @@
 button.version-switcher__button {
   border-color: var(--pst-color-border);
   color: var(--pst-color-text-base);
+  margin-bottom: 1em;
+
+  @include media-breakpoint-up($breakpoint-sidebar-primary) {
+    margin-bottom: none;
+  }
```

Comment:
```suggestion
  // Add a margin on narrow screens to avoid feeling cramped
  margin-bottom: 1em;
  @include media-breakpoint-up($breakpoint-sidebar-primary) {
    margin-bottom: unset;
  }
```